env/unique_id_gen.cc

```diff
@@ -189,7 +189,7 @@ void UnpredictableUniqueIdGen::GenerateNext(uint64_t* upper, uint64_t* lower) {
   // Use timing information (if available) to add to entropy. (Not a disaster
   // if unavailable on some platforms. High performance is important.)
 #ifdef __SSE4_2__  // More than enough to guarantee rdtsc instruction
-  extra_entropy = static_cast<uint64_t>(_rdtsc());
+  extra_entropy = static_cast<uint64_t>(__rdtsc());
```

Comment:
IIRC I tried `__rdtsc` and some compiler didn't like it, so I went with the documented Intel name. https://www.intel.com/content/www/us/en/docs/intrinsics-guide/index.html#text=rdtsc&ig_expand=5350 I haven't found any documentation for non-windows compilers using `__rdtsc`.

Yep: https://github.com/facebook/rocksdb/pull/11331/commits/ac260420113a99b985910811801c45dd9fe00f82

Also, if it breaks on Windows for you, can you update our CircleCI config(s) to cover a configuration that breaks on this code?

If we really need `__rdtsc` on Windows, I'd rather `#define _rdtsc() __rdtsc()` within the existing `#ifdef _WIN32` above

Reply:
Hello @pdillinger
Check my updates. I fix it with `#define` as you suggested. I also added additional Windows build with cpu target `AVX2`. Unfortunately, CircleCI didn't run this build.
I'm a little bit worried that my PR won't be enough. I saw PR: #11794 where @dawmster implement warning suppression for `crc32c.cc`